x-pack/plugins/cases/public/components/case_action_bar/index.tsx

```diff
@@ -43,20 +44,19 @@ interface CaseActionBarProps {
   caseData: Case;
   currentExternalIncident: CaseService | null;
   userCanCrud: boolean;
-  disableAlerting: boolean;
   isLoading: boolean;
   onRefresh: () => void;
   onUpdateField: (args: OnUpdateFields) => void;
 }
 const CaseActionBarComponent: React.FC<CaseActionBarProps> = ({
   caseData,
   currentExternalIncident,
-  disableAlerting,
   userCanCrud,
   isLoading,
   onRefresh,
   onUpdateField,
 }) => {
+  const { syncAlerts: syncAlertsFeatureOption } = useCasesFeatures();
```

Comment:
why this name change? Maybe we should be more specific in the hook return itself `const { syncAlertsFeature } = useCasesFeatures();` so we can keep the same name on all the places that is used.

Reply:
Because the same name variable is being used in a function below and the global variable of the hook overshadows the local one. I think the exporting name is fine (`syncAlerts`) as the hook's name is declarative enough (give me the sync alerts from the cases features).

Reply:
yes, the hook name is declarative, but in my oppinion the `syncAlerts` value is too generic, what about `isSyncAlertsEnabled`?

Reply:
Ok! I like it better.